Enhancement/10083 apply nullability phase 1

anayonkars opened the pull request:
Issue 10083: Apply Nullability to the project

This change is applicable only to `spring-integration-amqp`.
Replaced `org.springframework.lang.Nullable` with `org.jspecify.annotations.Nullable`.

<!--
Thanks for contributing to Spring Integration. 
Please provide a brief description of your pull-request and reference any related issue numbers (prefix references with #) or StackOverflow questions.

See the [Contributor Guidelines for more information](https://github.com/spring-projects/spring-integration/blob/main/CONTRIBUTING.adoc).
-->

anayonkars committed:
Issue 10083: Apply Nullability to the project

This change is applicable only to spring-integration-amqp.
Replaced org.springframework.lang.Nullable with org.jspecify.annotations.Nullable.

Signed-off-by: Anayonkar Shivalkar <anayonkar.shivalkar@gmail.com>

anayonkars committed:
Issue 10083: Apply Nullability to the project

Replaced org.springframework.lang.Nullable with org.jspecify.annotations.Nullable.

Signed-off-by: Anayonkar Shivalkar <anayonkar.shivalkar@gmail.com>

anayonkars committed:
Merge remote-tracking branch 'origin/enhancement/10083-Apply-Nullability-phase-1' into enhancement/10083-Apply-Nullability-phase-1

anayonkars committed:
Issue 10083: Apply Nullability to the project

This change is applicable only to spring-integration-amqp.
Fixed import order.

Signed-off-by: Anayonkar Shivalkar <anayonkar.shivalkar@gmail.com>

anayonkars committed:
Revert "Issue 10083: Apply Nullability to the project"

This reverts commit 079d57aeaec2580c4d73ce693ee3a4dfd643ae1f.

artembilan reviewed (changes requested):
```
 Error: eckstyle] [ERROR] /home/runner/work/spring-integration/spring-integration/spring-integration-amqp/src/main/java/org/springframework/integration/amqp/config/AmqpChannelFactoryBean.java:23:37: Using the '.*' form of import should be avoided - org.springframework.amqp.core.*. [AvoidStarImport]
Error: eckstyle] [ERROR] /home/runner/work/spring-integration/spring-integration/spring-integration-amqp/src/main/java/org/springframework/integration/amqp/config/AmqpChannelFactoryBean.java:49:1: Wrong order for 'java.util.Arrays' import. [ImportOrder]
Error: eckstyle] [ERROR] /home/runner/work/spring-integration/spring-integration/spring-integration-amqp/src/main/java/org/springframework/integration/amqp/outbound/AbstractAmqpOutboundEndpoint.java:49:1: Wrong order for 'java.time.Duration' import. [ImportOrder]
```
Please, use `gradlew check` command locally before pushing.
It will reveal all the Checkstyle violations as well, not only tests pass or so.

Thanks

anayonkars committed:
Issue 10083: Apply Nullability to the project

This change is applicable only to spring-integration-amqp.
Fixed import order.

Signed-off-by: Anayonkar Shivalkar <anayonkar.shivalkar@gmail.com>
